Dockerfile

```diff
@@ -1,27 +1,53 @@
-FROM fedora:latest
+# Arguments for DEV's (comment static FROM and uncomnnet #DEV ones)
```

Comment:
aha, got it, disregard comment on another #DEV line

I did not notice this line before

Reply:
I was thinking about two container files for images, finally I decided to just insert this comment.  I'm not sure if we want to leave it there or not, but it's useful from time to time.😄